vms/platformvm/txs/camino_add_validator_tx.go

```diff
@@ -37,7 +37,7 @@ func (tx *CaminoAddValidatorTx) Stake() []*avax.TransferableOutput {
 	return stake
 }
 
-// SyntacticVerify returns nil iff [tx] is valid
+// SyntacticVerify returns nil if [tx] is valid
```

Comment:
This and following files / changes should go at least into an own commit, optimal would be an own cleanup PR